metriken-exposition/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+use std::time::SystemTime;
```

Comment:
We don't have to do proper docs just yet but a small doc comment summarizing what this crate does would be useful to add here.